src/arrays.js

```diff
@@ -1,61 +1,66 @@
 const getNthElement = (index, array) => {
-  // your code here
+index %= array.length
+  return array[index];
+
 };
 
 const arrayToCSVString = array => {
-  // your code here
+return array.join();
 };
 
 const csvStringToArray = string => {
-  // your code here
+return string.split (',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+ return array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+return numbers.map(String);
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  return strings.map(string => string.toUpperCase());
 };
 
 const reverseWordsInArray = strings => {
-  // your code here
+  return strings.map(string => string.split("").reverse().join(""));
 };
 
 const onlyEven = numbers => {
-  // your code here
+  return numbers.filter(number => number % 2 === 0);
 };
 
 const removeNthElement2 = (index, array) => {
-  // your code here
+  const array2 = [...array];
+  array2.splice(index, 1);
+  return array2;
 };
 
 const elementsStartingWithAVowel = strings => {
-  // your code here
+  regex = /^[aeiou]/gi
```

Comment:
Well done using regex :)
There is only a little bit missing, which is declaring the variable regex.
To declare a variable you have to use declaration, const or let.
Const means constant, these are the variables which you do not want to reassign later on, and this is what is suggested to use as default. Let is used for variables to be reassigned later on, used it only when you are sure the variable is going to be reassigned later on. When a developer comes and reads your code and they see let, they will know that that variable is expected to be changed somewhere in the code, using let instead of const when the variable is not reassigned can cause confusion.

In this case you should just write:
`````
const regex = /^[aeiou]/gi
``````